hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -110,7 +103,6 @@
 
   covergroups: [
     {
-      //TODO Configuration for enable is currently not covered.
```

Comment:
The [CTRL register](https://opentitan.org/book/hw/ip/csrng/data/csrng.html#ctrl) has three fields: READ_INT_STATE, SW_APP_ENABLE and ENABLE. The last one is the overall module enable for CSRNG and I think this is still missing and should be added to the covergroup.

Reply:
Added a bin for `ctrl`.`enable` using the `enable` variable in `csrng_env_cfg.sv`